### ISSUE TITLE
AKEMI Audio not working after sof-setup-audio

### ISSUE THREAD

**Issue by BilalChouaibi** [cb-linux/breath#127]
[lsmod.txt](https://github.com/cb-linux/breath/files/8355236/lsmod.txt)
[after_sof-setup-audio.txt](https://github.com/cb-linux/breath/files/8355237/after_sof-setup-audio.txt)
[dmesg.txt](https://github.com/cb-linux/breath/files/8355238/dmesg.txt)
[find_usr_share_alsa.txt](https://github.com/cb-linux/breath/files/8355239/find_usr_share_alsa.txt)
[SOUNDCARD=rtk_sof-setup-audio.txt](https://github.com/cb-linux/breath/files/8355241/SOUNDCARD.rtk_sof-setup-audio.txt)

Lenovo Chromebook flex 5 Akemi
hi, my audio not working after execute your command. I test with all environment, debian and ubuntu. I also tested with and without updatekernel.sh but audio not working.

**Comment by nullexistence** [comment]
https://alsa-project.org/db/?f=faee43afec7095c720de273d4745de4b3a7b43c7
same issues

**Comment by MilkyDeveloper** [comment]
That’s odd. Does `speaker-test` work without running updatekernel? What’s the output of `cat $(which sof-setup-audio)`

**Comment by BilalChouaibi** [comment]
[speaker-test.txt](https://github.com/cb-linux/breath/files/8355690/speaker-test.txt)
[cat_command.txt](https://github.com/cb-linux/breath/files/8355691/cat_command.txt)
yes speaker-test return value

**Comment by MilkyDeveloper** [comment]
@BilalChouaibi, could you run `alsa-info` and give the link it creates or tell the output of the file it creates?

**Comment by BilalChouaibi** [comment]
[alsa-info.txt.ygMWBefjTo.txt](https://github.com/cb-linux/breath/files/8355735/alsa-info.txt.ygMWBefjTo.txt)

**Comment by MilkyDeveloper** [comment]
With `sudo`

**Comment by BilalChouaibi** [comment]
[alsa_output.txt](https://github.com/cb-linux/breath/files/8355741/alsa_output.txt)

**Comment by BilalChouaibi** [comment]
help

**Comment by BilalChouaibi** [comment]
Help please 😭

**Comment by MilkyDeveloper** [comment]
You have a Realtek soundcard. Try running `SOUNDCARD=rtk sof-setup-audio`.

**Comment by BilalChouaibi** [comment]
I already tried and it tells me done reboot but After reboot nothing working

**Comment by BilalChouaibi** [comment]
I also tried to reinstall everything and redo the command but nothing worked.

**Comment by BilalChouaibi** [comment]
I also tried to reinstall everything and redo the command but nothing worked.

**Comment by MilkyDeveloper** [comment]
After *rerunning* the RTK command, tell me the output of `sudo alsa-info`.

**Comment by BilalChouaibi** [comment]
[output_sudoals
[output2.txt](https://github.com/cb-linux/breath/files/8402942/output2.txt)
ainfo.txt](https://github.com/cb-linux/breath/files/8402940/output_sudoalsainfo.txt)

**Comment by MilkyDeveloper** [comment]
Could you provide the output of `aplay -l` and `dmesg` from within ChromeOS?

**Comment by BilalChouaibi** [comment]
[dmesg.txt](https://github.com/cb-linux/breath/files/8402975/dmesg.txt)
[aplay-l.txt](https://github.com/cb-linux/breath/files/8402976/aplay-l.txt)

**Comment by MilkyDeveloper** [comment]
What about `lsmod` from ChromeOS?

**Comment by BilalChouaibi** [comment]
[sudolsmod.txt](https://github.com/cb-linux/breath/files/8402996/sudolsmod.txt)
[lsmod.txt](https://github.com/cb-linux/breath/files/8402997/lsmod.txt)

**Comment by MilkyDeveloper** [comment]
In Breath, if you removed `/etc/systemd/system/alsa-reload.service`, ran `systemctl daemon-reload`, and rebooted, does audio work?

**Comment by BilalChouaibi** [comment]
okay i test it

**Comment by BilalChouaibi** [comment]
no it doesn't work

**Comment by MilkyDeveloper** [comment]
I am not sure how to fix this issue at the moment. What does `uname -r` output from ChromeOS?

**Comment by BilalChouaibi** [comment]
5.10.106

**Comment by MilkyDeveloper** [comment]
From ChromeOS

**Comment by BilalChouaibi** [comment]
4.19.225-12812-g959692546150

**Comment by Arawn-Davies** [comment]
I've also got an AKEMI Lenovo Flex 5 without working audio - the only thing that doesn't work. One thing I did remember seeing is that the mmcblk0 paths had to be updated to reference the NVMe storage to get the firmware files.

Sorry I haven't got any outputs or error messages at hand atm, but thought I should say about the NVMe path issue albeit very small


Thanks for great work! Makes it much more useful than ChromeOS :D
